org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corext/template/java/PostfixTemplate.java

```diff
@@ -83,6 +98,7 @@ class PostfixPreferences {
 	public static final String FORR_CONTENT = "for (int $${1:${index}} = ${i:inner_expression(array)}.length - 1; $${1:${index}} >= 0; $${1:${index}}--) {\n" +
 		"\t$${0}\n" +
 	"}";
+	public static final String FORMAT_CONTENT = "String.format(${i:inner_expression(java.lang.Object)}${});$${0}";
```

Comment:
Shouldn't `java.lang.Object` be `java.lang.String` instead ? `String.format(foo)` would only be valid for `foo` being a String. See https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/String.html#format(java.lang.String,java.lang.Object...) .